packages/ymax-planner/src/engine.ts

```
@@ -437,6 +536,72 @@ export const startEngine = async ({
       // TODO: Handle portfolio-level path `${VSTORAGE_PATH_PREFIX}.${portfolioKey}` for addition/update of depositAddress.
     }
 
+    // Handle subscriptions.
+    for (const { path, value: vstorageValue } of subscriptionEvents) {
+      const streamCell = tryJsonParse(
+        vstorageValue,
+        _err =>
+          Fail`non-JSON value at vstorage path ${q(path)}: ${vstorageValue}`,
```

should one corrupt record fail all the rest?  or should it be reported but we continue?

---

Right now it fails fast on the first bad record. I’m leaning towards logging/reporting the error and continuing so that one bad value doesn’t stop the whole stream. 

PS: I’m currently using `tryJsonParse` here to stay consistent with the `engine.ts` style.